bin/pbiviz.js

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env node
 /*
```

Comment:
As far as I know this row is required to allow execute pbiviz as a command line without npm